bridge/core/token.go

```diff
@@ -0,0 +1,246 @@
+package core
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"github.com/MichaelMure/git-bug/entity"
+	"regexp"
+	"strings"
+
+	"github.com/MichaelMure/git-bug/repository"
+)
+
+const (
+	tokenConfigKeyPrefix = "git-bug.token"
+	tokenValueKey        = "value"
+	tokenTargetKey       = "target"
+	tokenScopesKey       = "scopes"
+)
+
+// Token holds an API access token data
+type Token struct {
+	id     entity.Id
+	Value  string
+	Target string
+	Global bool
+	Scopes []string
```

Comment:
is storing the scopes really useful ?

Reply:
No not really, i thought we could use this just as a reminder to prompt when we suggest a token for a new bridge creation

Reply:
do you mind if we remove that ? It's not super useful and as scopes can be changed on the bugtracker side without git-bug knowing I'd like to avoid having git-bug displaying something incorrect.

Reply:
What might be useful is a creation date though, to quickly see the old token if you want to rotate them regularly, or just as a helper to remember where a token comes from.

Reply:
I like the idea of create date, i'll implement this and remove the scopes field